lib/github_changelog_generator/octo_fetcher.rb

```diff
@@ -95,7 +95,7 @@ def client
     # Fetch all tags from repo
     #
     # @return [Array <Hash>] array of tags
-    def get_all_tags
+    def fetch_all_tags
```

Comment:
Since this is a public API, even re-imagined by a neighbouring effort to use this codebase on GitLab repos, can we retain an alias to get_all_tags, to make it compatible? 

I like this rename, _and_ I'd like to make a compatibility alias.

Reply:
If you like, you can emit a deprecation warning in the get_all_tags method.

Reply:
Good catch. Brought back the old method with a deprecation warning.